cypress/integration/createOrganization.spec.js

```diff
@@ -70,25 +107,40 @@ describe("createOrganization", () =>{
         cy.get(organization.nextButton).click();
         cy.get(organization.nextButton).click();
         cy.get(organization.boardsPopUpOkButton).click({force: true});
+        cy.get(organization.scrumSign).should("be.visible").click();
+
+        cy.get(organization.titleField).should("be.visible").and("contain", name);
+
+        cy.get(organization.organizationList).eq(1).click();
         cy.get(organization.organizationSettings).eq(6).click();
         cy.get(organization.workingDaysCheckbox).eq(6).click();
         cy.get(organization.deleteOrganizationButton).scrollIntoView().click();
         cy.get(organization.passwordField).type(data.user.password);
         cy.get(organization.yesButton).click();
     })
 
-    it("updateOrganizationVacationDaysEmpty", () => {
+    it.only("updateOrganizationVacationDaysEmpty", () => {
         cy.get(organization.addOrganizationField).click();
         cy.get(organization.organizationNameField).type(name);
         cy.get(organization.nextButton).click();
         cy.get(organization.nextButton).click();
         cy.get(organization.boardsPopUpOkButton).click({force: true});
+        cy.get(organization.scrumSign).should("be.visible").click();
+
+        cy.get(organization.titleField).should("be.visible").and("contain", name);
+
+        cy.get(organization.organizationList).eq(1).click();
         cy.get(organization.organizationSettings).eq(6).click();
         cy.get(organization.vacationDaysField).clear();
         cy.get(organization.updateButton).eq(1).click();
+
+        cy.get(organization.vacationValidationMessage).eq(1).shoudl("be.visible").and("have.text", "The vacation days field is required");
+        
         cy.get(organization.deleteOrganizationButton).scrollIntoView().click();
         cy.get(organization.passwordField).type(data.user.password);
         cy.get(organization.yesButton).click();
+
+        cy.get(organization.titleField).should("be.visible").and("not.contain", name);
     })
 
     it("updateOrganizationVacationDays366", () => {
```

Comment:
Again, here is an example of a somewhat confusing description. I really can't tell what this test is supposed to do. Should it update the organization's vacation days to 366 (probably), or for 366 days, or maybe in?